Misc/NEWS.d/next/Core and Builtins/2021-04-22-10-46-40.bpo-43908.Co3YhZ.rst

```diff
@@ -0,0 +1,3 @@
+Introduce :const:`Py_TPFLAGS_IMMUTABLETYPE` for immutable type objects, and
```

Comment:
I suggest to move this NEWS entry to the C API category.

```suggestion
Introduce :const:`Py_TPFLAGS_IMMUTABLETYPE` flag for immutable type objects, and
```

Reply:
Yes, that makes sense.

Reply:
Fixed in 1987c4c027e41ae294711ca4c7d3ee9c905d5a7f